lib/puppet/pops/issues.rb

```diff
@@ -130,7 +135,7 @@ def self.hard_issue(issue_code, *args, &block)
   # @todo configuration
   #
   NAME_WITH_HYPHEN = issue :NAME_WITH_HYPHEN, :name do
-    "#{label.a_an_uc(semantic)} may not have a name containing a hyphen. The name '#{name}' is not legal"
+    "#{provider.a_an_uc(semantic)} may not have a name containing a hyphen. The name '#{name}' is not legal"
```

Comment:
We could just make `a_an`, `a_an_uc`, `the`, and `the_uc` available directly. Here that would be:
```
"#{a_an_uc(semantic)} may not have a name containing a hyphen. The name '#{name}' is not legal"
```